src/rgw/librgw.cc

```diff
@@ -1,5 +1,5 @@
 // -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
-// vim: ts=8 sw=2 smarttab
+// vim: ts=8 sw=2 smarttab ft=cpp
```

Comment:
these modelines came straight from https://github.com/ceph/ceph/blob/master/CodingStyle#L76-L82 so i'd prefer not to change them; why can't vim detect the filetype from the .cc extension?

Reply:
I'll update that too